dashboards/google-filestore/metadata.yaml

```diff
@@ -0,0 +1,10 @@
+sample_dashboards:
+  -
+    category: Filestore
+    id: filestore
```

Comment:
This should be the id of the sample_dashboard. In this case, that is `filestore_dashboard`.

Reply:
My bad, this comment is confusing. On line 1, the field should still be `sample_dashboards`. On this line, the id should be the name of the sample dashboard being added (i.e. `filestore_dashboard`).